cli/file_fetcher.rs

```diff
@@ -762,39 +739,6 @@ mod tests {
     assert_eq!(file.source, expected);
   }
 
-  #[test]
-  fn test_get_source_from_data_url() {
```

Comment:
Is there a reason to remove all these tests?
Are there any tests left that check that data URLs work properly?

Reply:
Yes. The removed tests check if data URLs are correctly processed. Now that we use the `data-url` crate, we no longer need them. And as far as the integration is concerned, the following two cases cover it.
https://github.com/denoland/deno/blob/f64630b8c152d9fb976c0ecdaa8abfff670932d8/cli/lsp/urls.rs#L188 and https://github.com/denoland/deno/blob/53414b4d3e6ef35e39ba8dacf8b406962ac99980/cli/file_fetcher.rs#L1001